src/utils/beforeBlockString.js

```diff
@@ -22,6 +22,12 @@ export default function (statement, { noRawBefore } = {}) {
   } else {
     result += "@" + statement.name + statement.raw("afterName") + statement.params
   }
-  result += statement.raw("between")
+
+  const between = statement.raw("between")
+
+  if (between) {
+    result += between
+  }
```

Comment:
This will not append `undefined` (is that the goal) but will also not append empty strings. I think we probably do want to append empty strings, right?